### DISCUSSION TITLE
Advanced Team Dynamics vs. Young Officer

### DISCUSSION THREAD

**Discussion by Felderburg** [bcholmes/StarTrek2d20#90] [Q&A]
The creator notes that the talent "Advanced Team Dynamics" (Player's Guide pg. 247) precludes a character being a Young Officer. I'm not sure why this is the case; as far as I can tell, the talent only has requirements that are somewhat unlikely for a Young Officer, but doesn't actively preclude that option outright.

**Comment by bcholmes** [comment]
I think the rule is that Advanced Team Dynamics is only available to Commanding officers (Captain, First Officer). I believe that it's also true that Young Officers cannot be those. But since the role is selected after all of the talents, it's tricksy to enforce.

**Reply by Felderburg** [reply]
I think the discussion about cadet being a rank, not a role, leading to the idea that a cadet can hold any role, indicates that a "young officer" could be CO or XO. The "untapped potential" talent does say a character can't be higher rank than lt. jg. with it, which makes being the CO or XO very unlikely, but not impossible.

I suppose a talent like this could tie in to a solution for #56, which involves taking talents prior to things about the character changing (in this case, role being defined) that make said talent unallowable.

I think a "simple character sheet" like the one that ships have may be an answer, where stats are put in and talents are just chosen.
